src/tools/compiletest/src/header.rs

```diff
@@ -874,6 +874,7 @@ pub fn make_test_description<R: Read>(
     let has_msan = util::MSAN_SUPPORTED_TARGETS.contains(&&*config.target);
     let has_tsan = util::TSAN_SUPPORTED_TARGETS.contains(&&*config.target);
     let has_hwasan = util::HWASAN_SUPPORTED_TARGETS.contains(&&*config.target);
+    let has_memtag = util::MEMTAG_SUPPORTED_TARGETS.contains(&&*config.target);
```

Comment:
This should also check that the test machine can actually run the tests using this sanitizer (i.e. supports MTE extension).

Would be good to also add at least one test for the feature.

Reply:
I'm not quite sure how to check for the presence of the build target supporting a target feature, could you provide a pointer? There isn't a target that has this enabled by default just yet.

Alternatively, I could as a test that checks the emitted assembly for the presence of tag-related instructions.

Reply:
Actually, I think its fine to leave it as is for now. It would be necessary if we wanted to actually run some code with this sanitizer, but we don't currently. And either way it would make sense to run all the necessary checks for the relevant cpu features during test's own runtime.